### DISCUSSION TITLE
Stream filter: follow returning all tweets at user instead of user's tweets

### DISCUSSION THREAD

**Discussion by ThomasH-Dev** [tweepy/tweepy#1552] [Questions] [answered]
So I am attempting to use the stream filter and I can't find quite enough details in Tweepy's documentation.

This line of code myStream.filter(track=['doge'], follow =["44196397"])  according to the documentation should return streamed tweets from the account specifically, however my stream has anyone that interacts with the account at all. Is this functioning correctly?
How can I include retweets from a specific account? 
How can I exclude terms?

Is there some more in depth documentation that I'm missing?

**Comment by Harmon758** [comment, marked as answer]
> This line of code myStream.filter(track=['doge'], follow =["44196397"]) according to the documentation should return streamed tweets from the account specifically, however my stream has anyone that interacts with the account at all. Is this functioning correctly?

If by "has anyone that interacts with the account at all", you mean replies to and Retweets of any Tweet from that account are included, yes. See the section on the `follow` parameter at https://developer.twitter.com/en/docs/twitter-api/v1/tweets/filter-realtime/guides/basic-stream-parameters.

> How can I include retweets from a specific account?

I'm not sure what you're asking.
Retweets are included in the stream.
If you want to add another account to follow, you can add another ID to the list you pass as the `follow` parameter.

> How can I exclude terms?

The standard Twitter API v1.1 POST statuses/filter endpoint stream doesn't support negation / NOT logic, so you'll have to filter them out yourself.

> Is there some more in depth documentation that I'm missing?

Tweepy's documentation is supplemented by the documentation for Twitter's API.
In this case, the documentation for the POST statuses/filter endpoint which `Stream.filter` uses is at https://developer.twitter.com/en/docs/twitter-api/v1/tweets/filter-realtime/api-reference/post-statuses-filter.
